### ISSUE TITLE
Clarify reasoning for font name changes

### ISSUE THREAD

**Issue by ryanoasis** [ryanoasis/nerd-fonts#136]
#### Clarify reasoning for font name changes

It is not clear enough (#135, #131) why some fonts names have been changed or appear to have typos.

These name changes are to comply with SIL Open Font License (OFL). Some fonts have parts of their name 'reserved', Fira for example: https://github.com/ryanoasis/nerd-fonts/blob/master/src/unpatched-fonts/FiraCode/LICENSE.txt

See the [SIL Open Font License](http://scripts.sil.org/OFL), in particular the [RFN section](http://scripts.sil.org/cms/scripts/page.php?item_id=OFL_web_fonts_and_RFNs#14cbfd4a):
> No Modified Version of the Font Software may use the Reserved Font
> Name(s) unless explicit written permission is granted by the corresponding
> Copyright Holder. This restriction only applies to the primary font name as
> presented to the users.

Although this is mentioned in the [patched fonts](https://github.com/ryanoasis/nerd-fonts#patched-fonts) table and the [Wiki FAQ](https://github.com/ryanoasis/nerd-fonts/wiki/FAQ#why-do-some-of-the-fonts-names-appear-incorrect-or-appear-to-have-typos) these are not easily found and/or clear enough.

---

Proposed solution:

1. Add a very clearly named text file to the release archive of the font to indicate a name change, e.g.:
`fira to fura.md`
   - in this text file describe the reasoning in greater detail
2. Update the font readmes in the patched fonts directory to include information on the name change, e.g.:
https://github.com/ryanoasis/nerd-fonts/tree/master/patched-fonts/FiraCode/Regular
make changes so that it appears similar to the following:

---

Fura Code Powerline
===================

Font creator
:   Nikita Prokopov

Version
:   1.102

Source
:   <https://github.com/tonsky/FiraCode>

License
:   SIL OPEN FONT LICENSE Version 1.1

Patched by
:   [Jamie Rolfs](https://github.com/jrolfs)

Fira Code is a Fira Mono font extended with a set of ligatures for
common programming multi-character combinations. This is just a font
rendering feature: underlying code remains ASCII-compatible. This helps
to read and understand code faster. For some frequent sequences like ..
or // ligatures allow us to correct spacing.

## Why `Fura` and not `Fira`?

What's in a name? The reason for the name change is to comply with the SIL Open Font License (OFL), in partcular the [Reserved Font Name mechanism](http://scripts.sil.org/cms/scripts/page.php?item_id=OFL_web_fonts_and_RFNs#14cbfd4a)

Some fonts have parts of their name "reserved" per the [Reserved Font Name mechanism](http://scripts.sil.org/cms/scripts/page.php?item_id=OFL_web_fonts_and_RFNs#14cbfd4a):
> No Modified Version of the Font Software may use the Reserved Font
> Name(s) unless explicit written permission is granted by the corresponding
> Copyright Holder. This restriction only applies to the primary font name as
> presented to the users.

- The main goals seem to be to: `Avoid collisions`, `Protect authors`, `Minimize support`, and `Encourage derivatives`

See the [Reserved Font Name section](http://scripts.sil.org/cms/scripts/page.php?item_id=OFL_web_fonts_and_RFNs#14cbfd4a) for additional information

### TL;DR

0. Pick your font family and then select from the `'complete'` directory.
  * Are you on Windows? Pick a font with the suffix `'Windows Compatible'`
  * Are you limited to mono fonts (because of your terminal, etc)? Pick a font with the suffix `'Mono'`

...
